src/settings.py

```diff
@@ -136,8 +136,8 @@ class GlobalSettings:
     grammar_search_true_pos_weight = 10
     grammar_search_false_pos_weight = 1
     grammar_search_bf_weight = 1
-    grammar_search_operator_size_weight = 1e-2
-    grammar_search_pred_complexity_weight = 1
+    grammar_search_operator_size_weight = 0.0
+    grammar_search_pred_complexity_weight = 1e-2
```

Comment:
i thought we were gonna do 1e-4, why change back?

Reply:
oops thank you